### ISSUE TITLE
Prevent vertical scrolling when scrolling horizontally

### ISSUE THREAD

**Issue by arthureberledev** [blaze-slider/blaze-slider#46]
When scrolling horizontally, vertical scrolling is disabled. But when scrolling vertically, horizontal is still possible. This leads to some weird scroll behavior on mobile devices as it can be seen in the video below.

Also, when doing this you get following error in console:
`[Intervention] Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.`

I first demonstrate the vertical scroll locking and afterwards the possible bug:
[screen-capture.webm](https://user-images.githubusercontent.com/58264696/217285037-4541b585-ebda-4ae0-9cc7-85bef30b55c4.webm)

**Comment by Chaciej** [comment]
Yes, I finished implementing it, was happy about just everything and then noticed this weird behavior, as you grab it and try to scroll vertical it starts scrolling horizontal but gets like 2-3 FPS while scrolling left-right together with document scrolling up-down, finally getting [Intervention] error

**Comment by arthureberledev** [comment]
> Yes, I finished implementing it, was happy about just everything and then noticed this weird behavior, as you grab it and try to scroll vertical it starts scrolling horizontal but gets like 2-3 FPS while scrolling left-right together with document scrolling up-down, finally getting [Intervention] error
> 
> Did you manage to fix it somehow?

Sadly not, i have replaced the slider for now with a different one.

**Comment by SuddenDev** [comment]
I have the same issue. 

@MananTank Do you have an idea how to fix that issue? I think we'd need a check if it's actually an horizontal swipe or a vertical scroll motion / swipe on mobile. Because this seems to break the entire thing on mobile devices.

Maybe I'll give it a shot later and put in a pr.
